recipes/arrow/all/conanfile.py

```diff
@@ -49,8 +49,9 @@ class ArrowConan(ConanFile):
         "with_glog": ["auto", True, False],
         "with_grpc": ["auto", True, False],
         "with_jemalloc": ["auto", True, False],
-        "with_mimalloc": ["auto", True, False],
+        "with_mimalloc": [True, False],
         "with_json": [True, False],
+        "with_thrift": ["auto", True, False],
```

Comment:
this option did not exist and was conditional to other one. I think it is clear to have it as a real option.